### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please don't use git clone or ldd.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+image: ghcr.io/capuanob/wsserver-fuzz:latest
```
Comment (xansec) [inline Mayhemfile:1]: Please omit image name. Tests has been changed to testsuite.

File: tests/fuzzy/Makefile [hunk 3]
```diff
@@ -23,16 +23,12 @@ LIB      =  $(WSDIR)/libws.a
 .PHONY: all run_fuzzy clean
```
Comment (xansec) [inline tests/fuzzy/Makefile:23]: Is this change necessary? Maybe you could make a new target `mayhem_fuzz` without deleting the initial target.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: tests/fuzzy/Makefile (hunk 3 above)
Comment (capuanob) [reply tests/fuzzy/Makefile:23]: Fixed, the issue was that run_fuzzy was running AFL fuzzing mid-build. So I took your advice and made a MAYHEM_FUZZ option

**Comment by capuanob** [comment]
@xansec Done!

**Review by xansec — APPROVED** [review]
LGTM!
